pkgs/servers/quagga/default.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv, fetchurl, libcap, libnl, readline, net_snmp, less, perl, texinfo }:
+
+stdenv.mkDerivation rec {
+  name = "quagga-${version}";
+  version = "0.99.24.1";
+
+  src = fetchurl {
+    url = "mirror://savannah/quagga/${name}.tar.gz";
+    sha256 = "0kvmc810m7ssrvgb3213271rpywyxb646v5bzjl1jl88vx3imbl4";
+  };
+
+  buildInputs =
+    [ readline net_snmp ]
+    ++ stdenv.lib.optionals stdenv.isLinux [ libcap libnl ];
+
+  nativeBuildInputs = [ perl texinfo ];
+
+  configureFlags = [
+    "--sysconfdir=/etc/quagga"
+    "--localstatedir=/run/quagga"
+    "--sbindir=$(out)/libexec/quagga"
+    "--disable-exampledir"
+    "--enable-user=quagga"
+    "--enable-group=quagga"
```

Comment:
Does quagga really need to have the user & group defined at compile time or is this this a default that can be overridden with command line arguments or a config file?